app/controllers/application_controller.rb

```diff
@@ -24,7 +24,7 @@ class ApplicationController < ActionController::Base
                 :tags,
                 :open_publisher
 
-  layout ->(c) { request.format == :mobile ? "application" : "centered_with_header_with_footer" }
+  layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
```

Comment:
Unused block argument - `c`. If it's necessary, use `_` or `_c` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.